paymentsheet/src/test/java/com/stripe/android/utils/FakeElementsSessionRepository.kt

```diff
@@ -36,7 +46,7 @@ internal class FakeElementsSessionRepository(
                     isGooglePayEnabled = isGooglePayEnabled,
                     sessionsError = sessionsError,
                     externalPaymentMethodData = externalPaymentMethodData,
-                    customer = customer,
+                    customer = this.customer,
```

Comment:
This name shadow seems error prone. We should probably change the names of these to make it more clear.